listening-platform/src/main/kotlin/com/nikitahohulia/listeningplatform/kafka/UserKafkaReceiver.kt

```diff
@@ -0,0 +1,28 @@
+package com.nikitahohulia.listeningplatform.kafka
+
+import com.nikitahohulia.api.internal.v2.usersvc.output.pubsub.update.proto.UserUpdatedEvent
+import com.nikitahohulia.listeningplatform.service.event.UserUpdatedEventService
+import jakarta.annotation.PostConstruct
+import org.springframework.stereotype.Component
+import reactor.core.scheduler.Schedulers
+import reactor.kafka.receiver.KafkaReceiver
+
+@Component
+class UserKafkaReceiver(
+    private val kafkaReceiverUserUpdatedEvent: KafkaReceiver<String, UserUpdatedEvent>,
+    private val userUpdatedEventService: UserUpdatedEventService<UserUpdatedEvent>
+) {
+
+    @PostConstruct
+    fun init() {
+        kafkaReceiverUserUpdatedEvent.receiveAutoAck()
+            .flatMap { fluxRecord ->
+                fluxRecord
+                    .map {
+                        userUpdatedEventService.publishEvent(it.value().user)
+                    }
+            }
+            .subscribeOn(Schedulers.boundedElastic())
```

Comment:
why `boundedElastic`?

Reply:
userUpdatedEventService.publishEvent(it.value().user) 
 
 Is a blocking operation, in addition as I understand, consuming messages can also be blocking, so as for me there is no reasons not to use ***boundedElastic***